raiden/network/channelgraph.py

```diff
@@ -232,6 +239,20 @@ def add_channel(self, details):
         channel_address = details.channel_address
         partner_state = details.partner_state
 
+        if details.settle_timeout < self.config['settle_timeout_min']:
+            log.info('ignoring channel {} because the settlement period is too low'.format(
+                pex(details.channel_address)
+            ))
+
+            return
+
+        if self.config['settle_timeout_max'] < details.settle_timeout:
```

Comment:
can you maybe reverse the condition here?
`details.settle_timeout > self.config['settle_timeout_max']`

Yoda conditions tend to be confusing to read.